HMCL/src/main/java/org/jackhuang/hmcl/ui/FXUtils.java

```diff
@@ -73,6 +70,7 @@
 import static org.jackhuang.hmcl.util.Lang.thread;
 import static org.jackhuang.hmcl.util.Lang.tryCast;
 
+@SuppressWarnings("rawtypes")
```

Comment:
You should reduce code affected by `@SuppressWarnings`. Only a single line can be annotated by `@SuppresWarnings`.